code/datums/materials/basemats.dm

```diff
@@ -269,3 +269,16 @@ Unless you know what you're doing, only use the first three numbers. They're in
 /datum/material/hot_ice/on_removed(atom/source, amount, material_flags)
 	qdel(source.GetComponent(/datum/component/hot_ice, "plasma", amount*150, amount*20+300))
 	return ..()
+
+/datum/material/metalhydrogen
+	name = "Metal Hydrogen"
+	id = "metal hydrogen"
+	desc = "Solid metallic hydrogen. Some say it should be impossible"
+	color = "#f2d5d7"
+	alpha = 150
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/metal_hydrogen
+	value_per_unit = 0.9
```

Comment:
You're going to want to consider a lower value than this. For reference, 0.1 is considered a commodity value for a datum materiam (Matched in price with plasma), and more than that is typically extremely limited materials like bananium or mythril.

And this is the first I'm seeing about hot ice so don't worry about that.

Reply:
this is metallic hydrogen, not hot ice. But yeah if you think is too high i can lower it a bit, even tho is an extremely limited material too